pkg/project/registry/projectrequest/delegated/delegated_test.go

```diff
@@ -1,6 +1,55 @@
 package delegated
 
-import "testing"
+import (
+	"testing"
+	"time"
 
-func TestDelegated(t *testing.T) {
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	authorizationclient "github.com/openshift/origin/pkg/authorization/client"
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+)
+
+func TestDelegatedWait(t *testing.T) {
```

Comment:
Suggestions for deflaking this test?   I didn't see a way to pass a fake clock into `PollImediate`